src/device-manager/python/Makefile.am

```diff
@@ -130,20 +130,20 @@ EXTRA_DIST +=										\
     $(NULL)
 
 .openweave-wheel: $(srcdir)/build-openweave-wheel.py weave-device-mgr _WeaveDeviceMgr.la $(shell find $(srcdir) -type f -name '*.py')
-	rm -f openweave-*.whl
+	rm -f *.whl
 	$(PYTHON) $(srcdir)/build-openweave-wheel.py
 	touch .openweave-wheel
 
 install-exec-local: .openweave-wheel
 	mkdir -p $(DESTDIR)$(pyexecdir)
-	rm -f $(DESTDIR)$(pyexecdir)/openweave-*.whl
-	cp openweave-*.whl $(DESTDIR)$(pyexecdir)
+	rm -f $(DESTDIR)$(pyexecdir)/*.whl
```

Comment:
This will delete all .whl files in the destination directory.  That could wipe out other unrelated .whl files.

Reply:
True, just assumed that there is only the openweave whl in that directory but we can probably restrict it to the specific package name.